components/threading/virtualized_thread_viewer/virtualized_thread_viewer.tsx

```diff
@@ -331,6 +364,34 @@ class ThreadViewerVirtualized extends PureComponent<Props, State> {
         return undefined;
     }
 
+    isNewMessagesVisible = (): boolean => {
+        const {visibleStopIndex, visibleStartIndex} = this.state;
+        const newMessagesSeparatorIndex = getNewMessageIndex(this.props.replyListIds);
+        if (typeof visibleStopIndex === 'number' && typeof visibleStartIndex === 'number') {
```

Comment:
Is this to differentiate 0 from undefined? Can we use `x == null` here?

If you are not aware about the `x == null` thing... http://adripofjavascript.com/blog/drips/equals-equals-null-in-javascript.html

Reply:
Did you plan to change this?

Reply:
Yes I did, the reason is that it showed the banner when scrolled between new messages line and bottom.
It should only show when above new messages line.